### PR TITLE
Some Editorial changes

### PR DESCRIPTION
**Pull request by billwuqin**
Add a few Editorial changes, in addition I think we should
1. Encourage collaboration with other venues such as MEF, TMF Multi-SDO initiative, ONUG for collaboration
2. Ambitious word is a little bit aggresive, can we replace with incentive?
3. We need to also consider PoC in some industry initiative such as ETSI ZSM, TMF catalyst project,  EANTC multi-vendor interoperability test event

### CODE REVIEW

**Review by mjethanandani — COMMENTED** [review]
Hi Qin, I feel the current description is in good shape and does not need further tweaks to take it to the IAB. Further edits can be had once IAB has had a chance to review the current description.

**Comment by billwuqin** [comment]
OK, I can hold my comments until it gets discussed in IAB.

**Comment by billwuqin** [comment]
Comparing NANOG, RIPE, APNIC, Autocon, I think TMF, MEF, ETSI ZSM don't represent pure operator community, may be not nessesary to be listed. So I tweak the proposal.

File: IAB Workshop.md [hunk 1]
```diff
@@ -51,17 +52,18 @@ Regarding operator deployments, discussion topics will include, but are not limi
...
 what new issues network operators and network management implementers are facing.
 
 The candidate items for discussion include, but are not limited to:
 
-- Tooling, opensource, experimentation, system integration
+- Tooling, opensource, experimentation, proof of concept, multi-vendor interoperability test (e.g., EANTC),
+  system integration
 
-- Data consistency to support richer observability (Data & Knowledge)
+- Data consistency to support richer full stack observability (Data & Knowledge)
```
Comment (boucadair) [inline IAB Workshop.md:66]:
```suggestion
- Data consistency to support richer observability (Data & Knowledge)
```

Not sure "full stack" will understood. I'm having troubles myself with parsing its meaning here.

File: IAB Workshop.md [hunk 2]
```diff
@@ -74,8 +76,9 @@ implementers, operational groups, and other entities.
 # Out of scope topics
 
 A workshop dedicated to management techniques related to encrypted
-networks has recently been held [RFC9490]; these topics are therefore out of
-scope for this workshop.
+networks has recently been held [RFC9490]; Another workshop dedicated
```
Comment (boucadair) [inline IAB Workshop.md:79]:
```suggestion
networks has recently been held [RFC9490]. Another workshop dedicated
```

File: IAB Workshop.md [hunk 3]
```diff
@@ -74,8 +76,9 @@ implementers, operational groups, and other entities.
...
+to environment impact of internet application and system has also been held
```
Comment (boucadair) [inline IAB Workshop.md:80]:
```suggestion
to environment impact of Internet applications and systems has also been held 
```

File: IAB Workshop.md [hunk 4]
```diff
@@ -74,8 +76,9 @@ implementers, operational groups, and other entities.
 # Out of scope topics
 
 A workshop dedicated to management techniques related to encrypted
-networks has recently been held [RFC9490]; these topics are therefore out of
-scope for this workshop.
+networks has recently been held [RFC9490]; Another workshop dedicated
+to environment impact of internet application and system has also been held 
+[RFC9547]. These topics are therefore out of scope for this workshop.
```
Comment (boucadair) [inline IAB Workshop.md:81]:
```suggestion
[RFC9547]. These topics are, therefore, out of scope for this workshop.
```

File: IAB Workshop.md [hunk 5]
```diff
@@ -51,17 +52,18 @@ Regarding operator deployments, discussion topics will include, but are not limi
 # Laying a path for the future of Network Management
 
 Network topologies have become significantly more complex since the
-2002 IAB workshop, as have router and host technologies. 
-Assuming this trend will continue, this workshop's ambition is
+2002 IAB workshop, as have the router and host technologies. 
+Assuming this trend will continue, this workshop's incentive is
```
Comment (boucadair) [inline IAB Workshop.md:56]:
```suggestion
Assuming this trend will continue, this workshop's ambition is
```

File: IAB Workshop.md [hunk 6]
```diff
@@ -13,7 +13,8 @@ recommendations for the IETF.
 Those requirements were instrumental in developing first the NETCONF
 protocol (in the NETCONF Working Group) [RFC6241], the associated YANG
 data modeling language (in the NETMOD Working Group) [RFC7950], 
-RESTCONF [RFC8040], and most recently CORECONF [draft-ietf-core-comi].
+RESTCONF [RFC8040], and most recently CORECONF [I.D-ietf-core-comi].
```
Comment (boucadair) [inline IAB Workshop.md:16]:
```suggestion
RESTCONF [RFC8040], and most recently CORECONF [I-D.ietf-core-comi].
```

**Review by boucadair — CHANGES_REQUESTED** [review]
Thanks. Some proposed edits.

**Pull request merged**